contracts/TablelandTables.sol

```diff
@@ -55,11 +55,14 @@ contract TablelandTables is
         payable
         override
         whenNotPaused
+        returns (uint256)
     {
         uint256 tableId = _nextTokenId();
         _safeMint(owner, 1);
 
         emit CreateTable(owner, tableId, statement);
+
+        return tableId;
```

Comment:
This is the meat of the feature.  In the case the caller is a contract, the tableId is available for the contract to use as needed